test/test_operations.py

```diff
@@ -1948,6 +1948,14 @@ def from_tensors(self, tensors):
     self.assertEqual(xdata.batch_sizes.device, torch.device('cpu'))
     self.assertEqual(xdata.data.device, xla_device)
 
+  def test_embedding_bag_backward(self):
```

Comment:
Hmm, the test does not do anything. Literally.
With LazyTensor nothing gets really executed there, modulo trips to CPU for eventually unsupported ops.
Do you have a test in PyTorch for this?

If not, maybe using the existing infrastructure, which checks against CPU:

https://github.com/pytorch/xla/blob/11ecce5a1f823d4c33f0f4e01232cff0a9e0147d/test/test_operations.py#L1023

Reply:
We are indeed testing trips to CPU for an unsupported op (EmbeddingBag). There is no test in PyTorch for this as it is only a problem for pytorch/XLA. Let me know what you think.

Reply:
You should be able to write a test like that, so we compare the results.

Reply:
Resolving. Test case removed.